tests/view_tests/tests/test_default_content_type.py

```diff
@@ -0,0 +1,44 @@
+import sys
```

Comment:
In c70ffeeb8b70b42693a340724f00eba656c6e51a you made me name the test file `test_removedindjango20`, by that logic this could be called `test_removedindjango30.py`